pkg/catalogsourceconfig/configuring.go

```diff
@@ -63,74 +61,31 @@ func (r *configuringReconciler) Reconcile(ctx context.Context, in *marketplace.C
 	// in case of a failure.
 	r.cache.Set(out)
 
-	err = r.reconcileCatalogSource(in)
+	csReconciler := catalogsource.NewResourcesReconciler(r.log, r.reader, r.client)
```

Comment:
This is a warping of the `reconciler` pattern. A `reconciler` is a type that takes an object associated with a particular controller and handles it by trying to reconcile between `spec` and `status`. We have reconcilers for objects that our operator handles (one per phase per custom resource). We don't `reconcile` these additional resources, we just create, update, or delete them.